### PR TITLE
Auto download enabled

### PR DESCRIPTION
**Pull request by vprusa**
Auto download enabled for Firefox and Chrome

### CODE REVIEW

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [208](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/208/)
<br> Gif records:<br> - [record_test_datasource.flv](http://10.16.23.184/records/208/record_test_datasource.flv) <br> - [record_test_deployments.flv](http://10.16.23.184/records/208/record_test_deployments.flv) <br> - [record_test_domains.flv](http://10.16.23.184/records/208/record_test_domains.flv) <br> - [record_test_providers.flv](http://10.16.23.184/records/208/record_test_providers.flv) <br> - [record_test_servers.flv](http://10.16.23.184/records/208/record_test_servers.flv) <br> - [record_test_topology.flv](http://10.16.23.184/records/208/record_test_topology.flv) <br>

File: common/session.py [hunk 1]
```diff
@@ -57,8 +58,40 @@ def __init__(self, login=True, add_provider=True, request = None):
...
-        self.logger.info("Using Browser: %s", self.BROWSER)
-        self.web_driver = getattr(webdriver,self.BROWSER)()
+        if self.BROWSER == "Firefox":
+            self.logger.info("Using Browser: %s", self.BROWSER)
+            profile = webdriver.FirefoxProfile()
+            # enable auto download
+            # http://stackoverflow.com/questions/24852709/how-do-i-automatically-download-files-from-a-pop-up-dialog-using-selenium-python
+            profile.set_preference("browser.download.folderList", 2)
+            profile.set_preference("browser.download.manager.showWhenStarting", False)
+            profile.set_preference("browser.download.dir", '~/Downloads/')
```
Comment (mattmahoneyrh) [inline common/session.py]: I question as to whether ~/Downloads will resolve properly.
Perhaps consider using something similar to the following, where the path resolves to the HOME environment variable:

"{}{}".format(os.getenv("HOME"),'/Downloads')

Comment (vprusa) [inline common/session.py]: I tested this both locally on Fedora and on selenium slave too and it was resolved correctly for both environments, for slave it was necessary to add ~/Downloads manually.

Comment (vprusa) [inline common/session.py]: Locally on Fedora under my user i did not have problem with ~/Downloads and files were saved correctly

**Pull request merged**